# This file is distributed under the same license as the Django package.
#
# Translators:
# Ivan Khomutov <iskhomutov@gmail.com>, 2017
# Jannis Leidel <jannis@leidel.info>, 2011
# Алексей Борискин <sun.void@gmail.com>, 2012-2015
# Андрей Щуров <shurov9009@gmail.com>, 2016
# Дмитрий Шатера <mr.bobsans@gmail.com>, 2016
msgid ""
msgstr ""
"Project-Id-Version: django\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-01-13 18:31+0200\n"
"PO-Revision-Date: 2017-09-30 20:31+0000\n"
"Last-Translator: Ivan Khomutov <iskhomutov@gmail.com>\n"
"Language-Team: Russian (http://www.transifex.com/django/django/language/"
"ru/)\n"
"Language: ru\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=4; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
"%10<=4 && (n%100<12 || n%100>14) ? 1 : n%10==0 || (n%10>=5 && n%10<=9) || (n"
"%100>=11 && n%100<=14)? 2 : 3);\n"

#: venv/lib/python3.7/site-packages/django/contrib/auth/admin.py:46
msgid "Personal info"
msgstr "Персональная информация"

#: venv/lib/python3.7/site-packages/django/contrib/auth/admin.py:47
msgid "Permissions"
msgstr "Права доступа"

#: venv/lib/python3.7/site-packages/django/contrib/auth/admin.py:49
msgid "Important dates"
msgstr "Важные даты"

#: venv/lib/python3.7/site-packages/django/contrib/auth/admin.py:133
#, python-format
msgid "%(name)s object with primary key %(key)r does not exist."
msgstr "%(name)s с первичным ключом %(key)r не существует."

#: venv/lib/python3.7/site-packages/django/contrib/auth/admin.py:143
msgid "Password changed successfully."
msgstr "Пароль успешно изменен."

#: venv/lib/python3.7/site-packages/django/contrib/auth/admin.py:163
#, python-format
msgid "Change password: %s"
msgstr "Изменить пароль: %s"

#: venv/lib/python3.7/site-packages/django/contrib/auth/apps.py:15
msgid "Authentication and Authorization"
msgstr "Пользователи и группы"

#: venv/lib/python3.7/site-packages/django/contrib/auth/base_user.py:48
msgid "password"
msgstr "пароль"

#: venv/lib/python3.7/site-packages/django/contrib/auth/base_user.py:49
msgid "last login"
msgstr "последний вход"

#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:31
msgid "No password set."
msgstr "Пароль не задан."

#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:36
msgid "Invalid password format or unknown hashing algorithm."
msgstr "Неизвестный формат пароля или алгоритм хеширования."

#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:71
#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:305
#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:378
msgid "The two password fields didn't match."
msgstr "Два поля с паролями не совпадают."

#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:74
#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:127
#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:163
#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:382
msgid "Password"
msgstr "Пароль"

#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:80
msgid "Password confirmation"
msgstr "Подтверждение пароля"

#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:83
#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:391
msgid "Enter the same password as before, for verification."
msgstr "Для подтверждения введите, пожалуйста, пароль ещё раз."

#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:129
msgid ""
"Raw passwords are not stored, so there is no way to see this user's "
"password, but you can change the password using <a href=\"{}\">this form</a>."
msgstr ""
"Пароли не хранятся в открытом виде, поэтому нет возможности увидеть пароль "
"этого пользователя, но вы можете изменить пароль, воспользовавшись<a href="
"\"{}\">этой формой</a>."

#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:170
#, python-format
msgid ""
"Please enter a correct %(username)s and password. Note that both fields may "
"be case-sensitive."
msgstr ""
"Пожалуйста, введите правильные %(username)s и пароль. Оба поля могут быть "
"чувствительны к регистру."

#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:173
msgid "This account is inactive."
msgstr "Эта учетная запись отключена."

#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:233
msgid "Email"
msgstr "Адрес электронной почты"

#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:308
msgid "New password"
msgstr "Новый пароль"

#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:314
msgid "New password confirmation"
msgstr "Подтверждение нового пароля"

#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:350
msgid "Your old password was entered incorrectly. Please enter it again."
msgstr "Ваш старый пароль введен неправильно. Пожалуйста, введите его снова."

#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:353
msgid "Old password"
msgstr "Старый пароль"

#: venv/lib/python3.7/site-packages/django/contrib/auth/forms.py:388
msgid "Password (again)"
msgstr "Пароль (еще раз)"

#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:260
#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:334
#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:430
#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:490
#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:521
#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:557
#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:593
#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:631
msgid "algorithm"
msgstr "алгоритм"

#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:261
msgid "iterations"
msgstr "итерации"

#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:262
#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:340
#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:432
#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:491
#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:522
#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:632
msgid "salt"
msgstr "соль"

#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:263
#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:341
#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:492
#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:523
#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:558
#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:594
#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:633
msgid "hash"
msgstr "хэш"

#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:335
msgid "variety"
msgstr "множество"

#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:336
msgid "version"
msgstr "версия"

#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:337
msgid "memory cost"
msgstr "затраты памяти"

#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:338
msgid "time cost"
msgstr "затраты времени"

#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:339
msgid "parallelism"
msgstr "параллелизм"

#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:431
msgid "work factor"
msgstr "рабочий фактор"

#: venv/lib/python3.7/site-packages/django/contrib/auth/hashers.py:433
msgid "checksum"
msgstr "контрольная сумма"

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:56
#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:111
msgid "name"
msgstr "имя"

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:60
msgid "content type"
msgstr "тип содержимого"

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:62
msgid "codename"
msgstr "кодовое название"

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:66
msgid "permission"
msgstr "право"

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:67
#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:114
msgid "permissions"
msgstr "права"

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:121
msgid "group"
msgstr "группа"

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:122
#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:218
msgid "groups"
msgstr "группы"

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:209
msgid "superuser status"
msgstr "статус суперпользователя"

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:212
msgid ""
"Designates that this user has all permissions without explicitly assigning "
"them."
msgstr "Указывает, что пользователь имеет все права без явного их назначения."

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:221
msgid ""
"The groups this user belongs to. A user will get all permissions granted to "
"each of their groups."
msgstr ""
"Группы, к которым принадлежит данный пользователь. Пользователь получит все "
"права, указанные в каждой из его/её групп."

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:229
msgid "user permissions"
msgstr "права пользователя"

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:231
msgid "Specific permissions for this user."
msgstr "Индивидуальные права данного пользователя."

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:298
msgid "username"
msgstr "имя пользователя"

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:301
msgid "Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only."
msgstr ""
"Обязательное поле. Не более 150 символов. Только буквы, цифры и символы @/./"
"+/-/_."

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:304
msgid "A user with that username already exists."
msgstr "Пользователь с таким именем уже существует."

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:307
msgid "first name"
msgstr "имя"

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:308
msgid "last name"
msgstr "фамилия"

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:309
msgid "email address"
msgstr "адрес электронной почты"

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:311
msgid "staff status"
msgstr "статус персонала"

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:313
msgid "Designates whether the user can log into this admin site."
msgstr ""
"Отметьте, если пользователь может входить в административную часть сайта."

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:316
msgid "active"
msgstr "активный"

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:319
msgid ""
"Designates whether this user should be treated as active. Unselect this "
"instead of deleting accounts."
msgstr ""
"Отметьте, если пользователь должен считаться активным. Уберите эту отметку "
"вместо удаления учётной записи."

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:323
msgid "date joined"
msgstr "дата регистрации"

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:332
msgid "user"
msgstr "пользователь"

#: venv/lib/python3.7/site-packages/django/contrib/auth/models.py:333
msgid "users"
msgstr "пользователи"

#: venv/lib/python3.7/site-packages/django/contrib/auth/password_validation.py:102
#, python-format
msgid ""
"This password is too short. It must contain at least %(min_length)d "
"character."
msgid_plural ""
"This password is too short. It must contain at least %(min_length)d "
"characters."
msgstr[0] ""
"Введённый пароль слишком короткий. Он должен содержать как минимум "
"%(min_length)d символ."
msgstr[1] ""
"Введённый пароль слишком короткий. Он должен содержать как минимум "
"%(min_length)d символа."
msgstr[2] ""
"Введённый пароль слишком короткий. Он должен содержать как минимум "
"%(min_length)d символов."
msgstr[3] ""
"Введённый пароль слишком короткий. Он должен содержать как минимум "
"%(min_length)d символов."

#: venv/lib/python3.7/site-packages/django/contrib/auth/password_validation.py:112
#, python-format
msgid "Your password must contain at least %(min_length)d character."
msgid_plural "Your password must contain at least %(min_length)d characters."
msgstr[0] "Ваш пароль должен содержать как минимум %(min_length)d символ."
msgstr[1] "Ваш пароль должен содержать как минимум %(min_length)d символа."
msgstr[2] "Ваш пароль должен содержать как минимум %(min_length)d символов."
msgstr[3] "Ваш пароль должен содержать как минимум %(min_length)d символов."

#: venv/lib/python3.7/site-packages/django/contrib/auth/password_validation.py:151
#, python-format
msgid "The password is too similar to the %(verbose_name)s."
msgstr "Введённый пароль слишком похож на %(verbose_name)s."

#: venv/lib/python3.7/site-packages/django/contrib/auth/password_validation.py:157
msgid "Your password can't be too similar to your other personal information."
msgstr ""
"Ваш пароль не должен совпадать с вашим именем или другой персональной "
"информацией или быть слишком похожим на неё."

#: venv/lib/python3.7/site-packages/django/contrib/auth/password_validation.py:185
msgid "This password is too common."
msgstr "Введённый пароль слишком широко распространён."

#: venv/lib/python3.7/site-packages/django/contrib/auth/password_validation.py:190
msgid "Your password can't be a commonly used password."
msgstr "Ваш пароль не может быть одним из широко распространённых паролей."

#: venv/lib/python3.7/site-packages/django/contrib/auth/password_validation.py:200
msgid "This password is entirely numeric."
msgstr "Введённый пароль состоит только из цифр."

#: venv/lib/python3.7/site-packages/django/contrib/auth/password_validation.py:205
msgid "Your password can't be entirely numeric."
msgstr "Ваш пароль не может состоять только из цифр."

#: venv/lib/python3.7/site-packages/django/contrib/auth/templates/registration/password_reset_subject.txt:2
#, python-format
msgid "Password reset on %(site_name)s"
msgstr "Сброс пароля на %(site_name)s"

#: venv/lib/python3.7/site-packages/django/contrib/auth/validators.py:12
msgid ""
"Enter a valid username. This value may contain only English letters, "
"numbers, and @/./+/-/_ characters."
msgstr ""
"Введите правильное имя пользователя. Это значение может содержать только "
"английские буквы, цифры и знаки @/./+/-/_."

#: venv/lib/python3.7/site-packages/django/contrib/auth/validators.py:22
msgid ""
"Enter a valid username. This value may contain only letters, numbers, and "
"@/./+/-/_ characters."
msgstr ""
"Введите правильное имя пользователя. Оно может содержать только буквы, цифры "
"и знаки @/./+/-/_."

#: venv/lib/python3.7/site-packages/django/contrib/auth/views.py:158
msgid "Logged out"
msgstr "Не авторизован"

#: venv/lib/python3.7/site-packages/django/contrib/auth/views.py:215
msgid "Password reset"
msgstr "Восстановление пароля"

#: venv/lib/python3.7/site-packages/django/contrib/auth/views.py:243
msgid "Password reset sent"
msgstr "Письмо с инструкциями по восстановлению пароля отправлено"

#: venv/lib/python3.7/site-packages/django/contrib/auth/views.py:252
msgid "Enter new password"
msgstr "Введите новый пароль"

#: venv/lib/python3.7/site-packages/django/contrib/auth/views.py:312
msgid "Password reset unsuccessful"
msgstr "Ошибка восстановления пароля"

#: venv/lib/python3.7/site-packages/django/contrib/auth/views.py:320
msgid "Password reset complete"
msgstr "Восстановление пароля завершено"

#: venv/lib/python3.7/site-packages/django/contrib/auth/views.py:332
msgid "Password change"
msgstr "Изменение пароля"

#: venv/lib/python3.7/site-packages/django/contrib/auth/views.py:355
msgid "Password change successful"
msgstr "Пароль успешно изменен"
